.github/workflows/scripts/e2e.delegator.default.verify.sh

```diff
@@ -39,6 +39,10 @@ echo "DEBUG: file is $THIS_FILE"
 
 export SLSA_VERIFIER_TESTING="true"
 
+echo "finding..................."
```

Comment:
TODO: remove these before merging. This script should not be used, it's for byob testing only.